package.json

```diff
@@ -31,7 +31,7 @@
     "pretest": "npm run lint",
     "test": "npm run jest",
     "jest": "cross-env NODE_ENV=test LOG_LEVEL=error MOCK_REDIS=1 FEED_URL_INTERVAL_MS=200 jest -c jest.config.js --",
-    "jest:e2e": "jest -c jest.config.e2e.js --",
+    "jest:e2e": "jest -c jest.config.e2e.js --runInBand --",
```

Comment:
`-i` works here as well

Reply:
I don't like using `-i` style flags in scripts, because you have no idea what they mean 6 months from now, and it's hard to search for on google, whereas doing a search for `jest runInBand` is easy.  When I'm typing commands over and over, the shorthand, single-letter version is great.  For scripts, it's almost always better to do full names.